app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -46,6 +46,8 @@ A bursary is paid over a minimum of 10 months by your teacher training provider
 
 You will not need to pay tax or National Insurance on bursary payments. 
 
+You do not have to pay a bursary back.
```

Comment:
Would it be more natural to say `pay back a bursary` rather than `pay a bursary back`?